client/src/components/PickupList.js

```diff
@@ -19,6 +34,21 @@ const PickupListPage: () => Node = () => {
     history.push("/customer-list");
   };
 
+  const AppointmentData = () => {
+    const {loading, error, data} = useQuery(FETCH_APPOINTMENTS);
+    
+    if(loading) return <tr><td>Loading...</td></tr>;
+    if(error) return <tr><td>Loading...</td></tr>;
+
+    return data.appointments.map( ({id, startTime, endTime, location, note}) => (
+        <tr key={id}>
+            <td>{startTime} - {endTime}</td>
+            <td>{location.address}</td>
+            <td>{note}</td>
```

Comment:
Can you keep the "confirmed customers" column (leave them empty for now)? We will need that in a later change.

Besides, I don't think we need to show the note column on this page as it's not essential to creating appointments and we want to keep the minimal amount of information on this page because the page is already quite complex.